utils/parts.stanza

```diff
@@ -238,13 +232,12 @@ public defn ComponentCode (json: JObject) -> ComponentCode :
     map(SupportCode, json-supports),
   )
 
-defn VendorPartNumbers (json: JObject) -> HashTable<String, VendorPartNumberCode> :
-  val vendor_part_numbers = HashTable<String, VendorPartNumberCode>()
+defn VendorPartNumbers (json: JObject) -> HashTable<String, String> :
+  val vendor_part_numbers = HashTable<String, String>()
   for key-value in entries(json) do:
     val vendor = key(key-value)
-    val vendor-part-object = value(key-value) as JObject
-    val vendor-part-number = vendor-part-object["value"] as String
-    vendor_part_numbers[vendor] = VendorPartNumberCode(true, vendor-part-number)
+    val vendor-part-number = value(key-value) as String
+    vendor_part_numbers[vendor] = vendor-part-number
   vendor_part_numbers
```

Comment:
Unsure where this is called, but except if you need to use it as an hashtable just afterwards, we prefer to use the immutable`Tuple<KeyValue<String, String>>`.

Otherwise with an hashtable this is just:
```
import lang-utils

defn VendorPartNumbers (json: JObject) -> HashTable<String, String> :
  to-hashtable(key, {value(_) as String}, entries(json))
```

With a tuple
```
defn VendorPartNumbers (json: JObject) -> Tuple<KeyValue<String, String>> :
  for kv in entries(json) map :
    key(kv) => (value(kv) as String)
```
or
```
defn VendorPartNumbers (json: JObject) -> Tuple<KeyValue<String, String>> :
  to-tuple $
    for [vendor, part-number] in kvs $ entries(json) seq :
      vendor => (part-number as String)
```

Reply:
Sorry it seems there is simpler in this case
```
defn VendorPartNumbers (json: JObject) -> Tuple<KeyValue<String, String>> :
  entries(json) as Tuple<KeyValue<String, String>>